src/test/java/com/shopify/pageObjects/SignUpPage.java

```diff
@@ -0,0 +1,78 @@
+package com.shopify.pageObjects;
+
+import com.shopify.utilityClasses.BrowserUtil;
+import org.openqa.selenium.*;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class SignUpPage {
+    WebDriver driver;
+    public SignUpPage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    // Locators
+    @FindBy(name = "customer[first_name]")
+    WebElement firstNameField;
+    public WebElement firstNameField(){
+        return firstNameField;
+    }
+
+    @FindBy(name = "customer[last_name]")
+    WebElement lastNameField;
+    public WebElement lastNameField(){
+        return lastNameField;
+    }
+
+
+    @FindBy(name = "customer[email]")
+    WebElement emailField;
+    public WebElement emailField(){
+        return emailField;
+    }
+
+
+    @FindBy(name = "customer[password]")
+    WebElement passwordField;
+    public WebElement passwordField(){
+        return passwordField;
+    }
```

Comment:
Webelements should not be returned by page objects instead only data should be returned